apps/frontend-pwa/src/pages/editProfile.tsx

```diff
@@ -145,85 +164,179 @@ function EditProfile() {
-                className="border-b-4 h-36 md:h-48 border-primary-80"
-                eyebrows="raised"
-                faceMask={false}
-                lashes={false}
-                mask={false}
-                clothing={values.clothing}
-                hat="none"
-                graphic="none"
-                accessory={values.accessory}
-                body="chest"
-                skinTone={values.skinTone}
-                clothingColor={values.clothingColor}
-                eyes={values.eyes}
-                hair={values.hair}
-                hairColor={values.hairColor}
-                mouth={values.mouth}
-                facialHair={values.facialHair}
-              />
+            <Form>
+              <div className="flex flex-col md:w-full md:max-w-5xl md:mx-auto gap-4">
+                <div className="flex flex-col w-full md:flex-row gap-4">
+                  <div className="flex flex-col justify-between flex-1 order-2 gap-3 md:order-1 md:bg-slate-50 md:p-4 rounded">
+                    <div>
+                      <H3 className={{ root: 'border-b mb-0' }}>
+                        {t('shared.generic.profile')}
+                      </H3>
+                      <div className="space-y-3 mb-2">
+                        <FormikTextField
+                          name="email"
+                          label={t('shared.generic.email')}
+                          labelType="small"
+                          className={{ label: 'font-bold text-md text-black' }}
+                        />
+                        <FormikTextField
+                          name="username"
+                          label={t('shared.generic.username')}
+                          labelType="small"
+                          className={{ label: 'font-bold text-md text-black' }}
+                        />
+                        <FormikTextField
+                          name="password"
+                          label={t('shared.generic.password')}
+                          labelType="small"
+                          className={{ label: 'font-bold text-md text-black' }}
+                          type="password"
+                        />
+                        <FormikTextField
+                          name="passwordRepetition"
+                          label={t('shared.generic.passwordRepetition')}
+                          labelType="small"
+                          className={{ label: 'font-bold text-md text-black' }}
+                          type="password"
+                        />
+                        <div>
+                          <div className="font-bold">
+                            {t('pwa.profile.publicProfile')}
+                          </div>
+                          <div className="flex flex-row space-between gap-4">
+                            <div className="flex flex-col items-center gap-1">
+                              <FormikSwitchField name="isProfilePublic" />
```

Comment:
new switch to toggle between self and others "Anonymous" and visibility of the pseudonyms